__test__/integration/adapters/collateralized_simple_interest_loan_adapter/scenarios/index.ts

```diff
@@ -0,0 +1,116 @@
+// External Libraries
+import { BigNumber } from "bignumber.js";
+import * as moment from "moment";
+
+// Utils
+import * as Units from "utils/units";
+
+// Wrappers
+import { DebtKernelContract } from "src/wrappers/contract_wrappers/debt_kernel_wrapper";
+import { RepaymentRouterContract } from "src/wrappers/contract_wrappers/repayment_router_wrapper";
+import { CollateralizedSimpleInterestTermsContractContract } from "src/wrappers/contract_wrappers/collateralized_simple_interest_terms_contract_wrapper";
+import { DummyTokenContract } from "src/wrappers/contract_wrappers/dummy_token_wrapper";
+
+// Adapters
+import { SimpleInterestTermsContractParameters } from "src/adapters/simple_interest_loan_adapter";
+import { CollateralizedTermsContractParameters } from "src/adapters/collateralized_simple_interest_loan_adapter";
+
+// Types
+import { DebtOrder } from "src/types/debt_order";
+
+export interface ReturnCollateralScenario {
+    // The test's description.
+    description: string;
+    // True if the scenario succeeds in returning collateral.
+    succeeds: boolean;
+    simpleTerms: SimpleInterestTermsContractParameters;
+    collateralTerms: CollateralizedTermsContractParameters;
+    // A function that returns a valid DebtOrder instance.
+    generateDebtOrder: (
+        debtKernel: DebtKernelContract,
+        repaymentRouter: RepaymentRouterContract,
+        principalToken: DummyTokenContract,
+        termsContract: CollateralizedSimpleInterestTermsContractContract,
+        termsParams: string,
+    ) => DebtOrder.Instance;
+    // Typically the underwriter fills the debt order.
```

Comment:
This is not typically the case -- more of an unusual case if anything.  Please fix comment to clarify.